src/icrar/leap-accelerate-cli/main.cc

```diff
@@ -242,7 +96,8 @@ int main(int argc, char** argv)
     icrar::profiling::UsageReporter _;
     try
     {
-        ArgumentsValidated args = ArgumentsValidated(std::move(rawArgs));
+        ArgumentsValidated args = ArgumentsValidated(std::move(Arguments(std::move(rawArgs))));
```

Comment:
`ArgumentsValidated args {Arguments{std::move(rawArgs)}};`, no need for the second `move` (but you need list-initialization syntax to avoid the most vexing parse).